### DISCUSSION TITLE
Geomap Geojson Layer shape labels visible

### DISCUSSION THREAD

**Discussion by tedraymond** [grafana/grafana#57030] [Dashboards, Panels & Visualizations]
Is there any information on adding layer title directly in the map view, or toggle the layer title on and off. Much like markers but for geojson layers

**Comment by maurerle** [comment]
I was asking this myself too.

The sample geojson has a name included as properties - but it is not shown when hovering above the geojson:
![image](https://user-images.githubusercontent.com/25026204/219307649-f85af4b7-87ca-40ae-a9d1-552d5c09b526.png)

https://github.com/grafana/grafana/blob/main/public/maps/usa-states.geojson?short_path=c492d1b

I would like to have a map with data links to other Panels as an intro view of our dashboard

EDIT: One can enable the tooltip, which then shows the information for each shape

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
